README.md

```diff
@@ -84,7 +84,9 @@ VCR_RECORD_MODE=all bin/rake
 
 The deployment is triggered on all builds in [CircleCI](https://circleci.com/gh/ministryofjustice/laa-apply-for-legal-aid) but requires approval to the desired environment.
 
-**NOTE:** **git-crypt** is required to store secrets required for **uat**, **staging** and **production** environments. To be able to modify those secrets, **git-crypt** needs to be set up according to the following [guide](https://ministryofjustice.github.io/cloud-platform-user-docs/03-other-topics/001-git-crypt-setup/#git-crypt).
+**NOTE:** **git-crypt** is required to store secrets required for **uat**, **staging** and **production** environments.
+To be able to modify those secrets, **git-crypt** needs to be set up according to the following
+[guide](https://ministryofjustice.github.io/cloud-platform-user-docs/03-other-topics/001-git-crypt-setup/#git-crypt).
```

Comment:
Just tidying up (spitting the text into separate lines so it's easier to read within an editor)